### PR TITLE
Add temporary table for PositionUser

### PR DESCRIPTION
**Pull request by SolovyovDmitry**

### CODE REVIEW

File: src/PositionService.Data.Provider.MsSql.Ef/Migrations/20211021130000_InitialMigration.cs [hunk 1]
```diff
@@ -43,14 +43,23 @@ private void CreatePositionUsersTable(MigrationBuilder migrationBuilder)
...
+            .Annotation("SqlServer:TemporalPeriodEndColumnName", "PeriodEnd")
+            .Annotation("SqlServer:TemporalPeriodStartColumnName", "PeriodStart"),
+          PeriodStart = table.Column<DateTime>(type: "datetime2", nullable: false)
+            .Annotation("SqlServer:IsTemporal", true)
+            .Annotation("SqlServer:TemporalPeriodEndColumnName", "PeriodEnd")
+            .Annotation("SqlServer:TemporalPeriodStartColumnName", "PeriodStart")
         },
         constraints: table =>
         {
           table.PrimaryKey("PK_PositionUsers", p => p.Id);
```
Comment (MariyaSin) [inline src/PositionService.Data.Provider.MsSql.Ef/Migrations/20211021130000_InitialMigration.cs:57]: rename to "PositionsUsers"

File: src/PositionService.Models.Db/DbPositionUser.cs [hunk 2]
```diff
@@ -8,15 +8,13 @@ namespace LT.DigitalOffice.PositionService.Models.Db
   public class DbPositionUser
   {
     public const string TableName = "PositionUsers";
```
Comment (MariyaSin) [inline src/PositionService.Models.Db/DbPositionUser.cs:10]: "PositionsUsers"

File: src/PositionService.Data/PositionUserRepository.cs [hunk 3]
```diff
@@ -46,6 +47,30 @@ public async Task<List<DbPositionUser>> GetAsync(List<Guid> userIds)
         .ToListAsync();
     }
 
+    public async Task<Guid?> EditAsync(Guid userId, Guid? positionId)
```
Comment (MariyaSin) [inline src/PositionService.Data/PositionUserRepository.cs:50]: edit method must have value, use remove method if positionId have not value

File: src/PositionService.Mappers/Data/PositionUserDataMapper.cs [hunk 4]
```diff
@@ -13,7 +14,7 @@ public PositionUserData Map(DbPositionUser user)
         return null;
       }
 
-      return new(user.UserId, user.CreatedAtUtc);
+      return new(user.UserId, DateTime.Now);
```
Comment (MariyaSin) [inline src/PositionService.Mappers/Data/PositionUserDataMapper.cs:17]: add TO DO to fix it

**Comment by vorobevamv** [comment]
update libraries, please (update Kernel to 1.1.61 and add Kernel.EFSupport):

1. in Common.Configurations.targets
1.1 change
<PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.59" />
to
<PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.61" />

1.2 add
<PackageReference Include="LT.DigitalOffice.Kernel.EFSupport" Version="1.0.1" />

1.3 delete (lines 33-38)
  <ItemGroup Condition="($(AssemblyName.Contains('.Provider')) Or $(AssemblyName.Contains('.Models.Db'))) And $(AssemblyName.Contains('.UnitTests')) != 'true'">
    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.0" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="6.0.0" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.0" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.0" />
  </ItemGroup>

2.in IDataProvider.cs
change
using LT.DigitalOffice.Kernel.Database;
to
using LT.DigitalOffice.Kernel.EFSupport.Provider;

3. in Startup.cs
3.1 add
using LT.DigitalOffice.Kernel.EFSupport.Extensions;
using LT.DigitalOffice.Kernel.EFSupport.Helpers;

3.2 change (lines 198-208)
      string connStr = Environment.GetEnvironmentVariable("ConnectionString");
      if (string.IsNullOrEmpty(connStr))
      {...}
      else
      {...}
to
 string connStr = ConnectionStringHandler.Get(Configuration);

3.3 change (line 241)
UpdateDatabase(app);
to
app.UpdateDatabase<PositionServiceDbContext>();

3.4 delete method (lines 124-133)
private void UpdateDatabase(IApplicationBuilder app)
{...}

4. don't forget to do rebuild and clean usings

**Pull request merged**